### PR TITLE
Duplication of the `0.22.1` release

### PR DESCRIPTION
**Pull request by xgreenx**
It is a duplication of the https://github.com/FuelLabs/fuel-core/pull/1664 for the `master`.

### CODE REVIEW

**Review by bvrooman — APPROVED** [review]

**Comment by bvrooman** [comment]
I see Mitch had an opinion about the fix: https://github.com/FuelLabs/fuel-core/pull/1664#pullrequestreview-1879194363. 
I won't block this PR, but I suggest that you come to an agreement about the comment before merging this.

**Review by Voxelot — APPROVED** [review]

**Pull request merged**
